packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts

```diff
@@ -101,6 +102,12 @@ export interface LoadBalancedFargateServiceProps {
    * Setting this option will set the load balancer port to 443.
    */
   certificate?: CertificateRef;
+  /**
+   * Allows opting out of creation of aws log drivers
+   *
+   * @default true
+   */
+  createLogDriver?: boolean;
```

Comment:
Should we mention it's the AWS one?

Reply:
"Log driver" feels too low level. I can know how to run a docker image locally without having any idea what log drivers are. Maybe just "logging"?

Reply:
I had `logging` originally but didn't want that to be confused with the `logging` field on the `ContainerDefinition` construct. I think `enableLogs` is better but wanted to be explicit that we're actually creating a LogGroup resource.